model/bin/ww3_from_ftp.sh

```diff
@@ -53,6 +53,7 @@ cp -r data_regtests/ww3_tp2.14/input/toy/toy_coupled_field.nc.OASACM regtests/ww
 cp -r data_regtests/ww3_tp2.14/input/toy/*.nc regtests/ww3_tp2.14/input/toy/
 cp -r data_regtests/ww3_tp2.17/input/*     regtests/ww3_tp2.17/input/
 cp -r data_regtests/ww3_tp2.21/input/*     regtests/ww3_tp2.21/input/
+cp -r data_regtests/ww3_test_09/input/*    regtests/mww3_test_09/input/
```

Comment:
please correct it:
cp -r data_regtests/mww3_test_09/input/* regtests/mww3_test_09/input/